### ISSUE TITLE
Forking/cloning

### ISSUE THREAD

**Issue by kallewesterling** [DHRI-Curriculum/git#18]
In [What You Can Do with Git and GitHub](https://github.com/DHRI-Curriculum/git/blob/master/sections/examples.md) we might want to make sure that `forking` is explained as a term, especially because we don't explain the term later in the workshop either.

**Comment by hackettka** [comment]
Good point. I would also like to clean up some of the workshop a bit this summer. Having taught it two times, I think there is defn clarification I can add throughout that would make this easier to use and to teach. Just as an FYI.

**Comment by kallewesterling** [comment]
I agree - this is great stuff for future sprints! Let's add a label here to keep it on our radar.

**Comment by hackettka** [comment]
i think this has been addressed. I added some clarification towards the beginning where it is used; and i added a short section at the end about what forking is and why you might use it. its included in the same section as cloning, and how and why you might use them together is also included. they then proceed through a challenge of forking our git  & github workshop repo, and cloning it, that also asks them to make changes and send them to github.
